monitor/main.go

```diff
@@ -352,6 +356,14 @@ func blockProcessor(ctx context.Context, headers <-chan *types.Header, cc *clien
 				validatorsProcessor.HandleLog(eventLog)
 				sortedOraclesProcessor.HandleLog(eventLog)
 			}
+
+			internalTransfers, err := cc.Debug.TransactionTransfers(ctx, txHash)
+			if err != nil {
+				return err
+			}
+			for _, internalTransfer := range internalTransfers {
+				logTransaction(txLogger, "method", "transferCGLD", "from", internalTransfer.From, "to", internalTransfer.To, "value", internalTransfer.Value.Uint64())
```

Comment:
I would create a separate `logTransfer` method here as that is different from a transaction which has a different meaning. Also value might exceed the bounds of uint64, so we should just return it as a string here and do the processing downstream